electronics_model/test_kicad_import.py

```diff
@@ -0,0 +1,90 @@
+
+        expected_conn = edgir.ValueExpr()
+        expected_conn.connected.link_port.ref.steps.add().name = 'Test_Net_1'
+        expected_conn.connected.link_port.ref.steps.add().name = 'passives'
+        expected_conn.connected.link_port.ref.steps.add().allocate = ''
+        expected_conn.connected.block_port.ref.steps.add().name = 'C1'
+        expected_conn.connected.block_port.ref.steps.add().name = 'neg'
+        self.assertIn(expected_conn, self.pb.constraints.values())
+
+        expected_conn = edgir.ValueExpr()
+        expected_conn.connected.link_port.ref.steps.add().name = 'Test_Net_1'
+        expected_conn.connected.link_port.ref.steps.add().name = 'passives'
+        expected_conn.connected.link_port.ref.steps.add().allocate = ''
+        expected_conn.connected.block_port.ref.steps.add().name = 'R2'
+        expected_conn.connected.block_port.ref.steps.add().name = 'b'
+        self.assertIn(expected_conn, self.pb.constraints.values())
+
+        expected_conn = edgir.ValueExpr()
+        expected_conn.connected.link_port.ref.steps.add().name = 'Net-(C1-Pad1)'
+        expected_conn.connected.link_port.ref.steps.add().name = 'passives'
+        expected_conn.connected.link_port.ref.steps.add().allocate = ''
+        expected_conn.connected.block_port.ref.steps.add().name = 'R1'
+        expected_conn.connected.block_port.ref.steps.add().name = 'b'
+        self.assertIn(expected_conn, self.pb.constraints.values())
+
+        expected_conn = edgir.ValueExpr()
+        expected_conn.connected.link_port.ref.steps.add().name = 'Net-(C1-Pad1)'
+        expected_conn.connected.link_port.ref.steps.add().name = 'passives'
+        expected_conn.connected.link_port.ref.steps.add().allocate = ''
+        expected_conn.connected.block_port.ref.steps.add().name = 'C1'
+        expected_conn.connected.block_port.ref.steps.add().name = 'pos'
+        self.assertIn(expected_conn, self.pb.constraints.values())
+
+        expected_conn = edgir.ValueExpr()
+        expected_conn.connected.link_port.ref.steps.add().name = 'PORT_A_link'
+        expected_conn.connected.link_port.ref.steps.add().name = 'passives'
+        expected_conn.connected.link_port.ref.steps.add().allocate = ''
+        expected_conn.connected.block_port.ref.steps.add().name = 'R2'
+        expected_conn.connected.block_port.ref.steps.add().name = 'a'
+        self.assertIn(expected_conn, self.pb.constraints.values())
+
+        expected_conn = edgir.ValueExpr()
+        expected_conn.connected.link_port.ref.steps.add().name = 'PORT_A_link'
+        expected_conn.connected.link_port.ref.steps.add().name = 'passives'
+        expected_conn.connected.link_port.ref.steps.add().allocate = ''
+        expected_conn.connected.block_port.ref.steps.add().name = 'R1'
+        expected_conn.connected.block_port.ref.steps.add().name = 'a'
+        self.assertIn(expected_conn, self.pb.constraints.values())
+
+        expected_conn = edgir.ValueExpr()
```

Comment:
For simplicity and to keep tests orthogonal and independent it may be better to have a circuit where a bridge doesn't need to be inferred - practically this means that only one pin is connected to a boundary port (PORT_A here)